rdagent/scenarios/data_science/dev/runner/eval.py

```diff
@@ -52,19 +52,23 @@ def evaluate(
             env=env, entry=get_clear_ws_cmd()
         )  # Remove previous submission and scores files generated by worklfow.
 
-        # execute workflow
-        result = implementation.run(env=env, entry="python -m coverage run main.py")
-        stdout = result.stdout
-        execute_ret_code = result.exit_code
-        implementation.running_info.running_time = result.running_time
-
-        match = re.search(r"(.*?)=== Start of EDA part ===(.*)=== End of EDA part ===", stdout, re.DOTALL)
-        eda_output = match.groups()[1] if match else None
-        if eda_output is None:
-            eda_output = "No EDA output."
-        implementation.inject_files(**{"EDA.md": eda_output})
-        stdout = remove_eda_part(stdout)
-        stdout += f"The code executed {'successfully' if execute_ret_code == 0 else 'failed'}. {'The EDA output is removed from the stdout. ' if eda_output else ''}"
+        if DS_RD_SETTING.sample_data:
+            # execute workflow
+            result = implementation.run(env=env, entry="python -m coverage run main.py")
+            stdout = result.stdout
+            execute_ret_code = result.exit_code
+            implementation.running_info.running_time = result.running_time
+
+            match = re.search(r"(.*?)=== Start of EDA part ===(.*)=== End of EDA part ===", stdout, re.DOTALL)
+            eda_output = match.groups()[1] if match else None
+            if eda_output is None:
+                eda_output = "No EDA output."
+            implementation.inject_files(**{"EDA.md": eda_output})
+            stdout = remove_eda_part(stdout)
+            stdout += f"The code executed {'successfully' if execute_ret_code == 0 else 'failed'}. {'The EDA output is removed from the stdout. ' if eda_output else ''}"
+        else:
+            execute_ret_code = 0
+            stdout = "The code executed successfully in the coder stage. No need to check the code execution in the runner stage. Please only focus on the metrics and submission files."
 
         # Check score file
         score_fp = implementation.workspace_path / "scores.csv"
```

Comment:
If we not sample_data,
How can we ensure the "scores.csv" correctly generated ?